data-model/src/main/scala/za/co/absa/enceladus/model/menas/HDFSFolder.scala

```diff
@@ -15,9 +15,9 @@
 
 package za.co.absa.enceladus.model.menas
 
-case class HDFSFolder
+case class HDFSFolder2
```

Comment:
Why HDFSFolder2? Shouldn't it be deleted?

Reply:
Good catch